frontend/components/pages/CourseList.tsx

```diff
@@ -59,22 +61,32 @@ export function CourseList({ courses, handleTextChange }: CourseListProps) {
             </InputRightElement>
           </InputGroup>
         </Box>
-        <Box mb={10} minW={'1024px'}>
-          <Flex
-            alignItems="center"
-            pb={2}
-            borderBottom="1px"
-            borderColor={'blackAlpha.500'}
-          ></Flex>
-          {courses.length === 0 && (
-            <VStack mt="40px">
-              <Heading color={PRIMARY_FONT_COLOR} fontSize="36px">
-                検索に一致するコースはありませんでした
-              </Heading>
-            </VStack>
-          )}
-          {courses && <CourseCard courses={courses} />}
-        </Box>
+        <Flex>
+          <Box mb={10} minW={'880px'}>
+            <Flex
+              alignItems="center"
+              pb={2}
+              borderBottom="1px"
+              borderColor={'blackAlpha.500'}
+            ></Flex>
```

Comment:
```suggestion
          <Box mb={10} minW={'880px'}>
          ```suggestion
        <Flex>
        <Heading py={10} color={PRIMARY_FONT_COLOR} fontSize="36px">
          コース一覧
        </Heading>
        <Box marginLeft={'auto'}>
          <InputGroup>
            <Input
              value={text}
              onChange={(e) => handleInputChange(e)}
              onKeyDown={(e) => handleKeyDown(e)}
              variant="outline"
              placeholder="javaScript"
              background={'white'}
            />
            <InputRightElement onClick={() => handleSubmit()}>
              <SearchIcon color="gray.500" />
            </InputRightElement>
          </InputGroup>
        </Box>
            <Flex
              alignItems="center"
              pb={2}
              borderBottom="1px"
              borderColor={'blackAlpha.500'}
            ></Flex>
```

みたいな形で
- コース一覧
- 検索ボックス

も画面左のカラムに入れたいですね！！